tools/docker/Dockerfile.integ-test

```diff
@@ -30,18 +30,30 @@ RUN mkdir -p \
   ${FICD_LOG_DIR} \
   /etc/cni/net.d
 
-# Install containerd to have runc shim.
-ENV CTRD_VERSION="1.6.4"
+# Install containerd to have runc shim for amd.
```

Comment:
The install for ARM will overwrite the x86 shim binary, which causes failures in tests requiring this binary run on x86 machines. Let's distinguish between the architecture this is running on so we download and install the proper binary.

```suggestion
# Install containerd to have runc shim.
ENV CTRD_VERSION="1.6.8"
RUN ARCH=`uname -m` && \
  if [ "$ARCH" == "x86_64" ]; then \
      ARCH="amd64"; \
  elif [ "$ARCH" == "aarch64" ]; then  \
      ARCH="arm64"; \
  fi  && \
  wget --quiet -O- https://github.com/containerd/containerd/releases/download/v$CTRD_VERSION/containerd-$CTRD_VERSION-linux-$ARCH.tar.gz | tar zxf - -C /tmp/ && \
  install -D -o root -g root -m755 -t /usr/local/bin /tmp/bin/containerd-shim-runc-v2 && \
  rm -rf /tmp/bin

```

Reply:
Noted, thanks for the suggestion